app/views/events/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1 class="text-center">event details</h1>
+<main class="container">
+        <div class="mx-auto  border border-success events">
+                <div class="border">
+                        <h1 class="text-center"><%= @event.event_title %></h1>
+                        <p><%= @event.description %></p>
```

Comment:
If the description is too long we get this:

![image](https://user-images.githubusercontent.com/22228397/64927859-dbf84e80-d7d5-11e9-9aca-c17b967d17ee.png)

[OPTIONAL]